test/metabase/models/session_test.clj

```diff
@@ -129,3 +129,17 @@
                                                                        :embedded           false
                                                                        :ip_address         "0:0:0:0:0:0:0:1"})
           (is (true? @email-sent)))))))
+
+(deftest clean-sessions-test ()
+  (mt/with-temp [:model/User {user-id :id} {}
+                 :model/Session old-session {:id         "a"
+                                             :user_id    user-id
+                                             :created_at (t/minus (t/local-date) (t/months 1))}
```

Comment:
It might be nicer to do:

```
(mt/with-temp-env-var-value! [mb-max-session-age (* 60 24)] ;; one day
  (mt/with-temp [:model/Session old-session {:created_at (t/minus (t/local-date) (t/days 2))])
```

to a) show that you're respecting that config value, and b) not mess up local tests for someone who's locally modified `MB_MAX_SESSION_AGE` for some crazy reason.

Reply:
Good idea. Fixed it